uint8 isFind
leaf_msg[] res
